DiscordBot/Commands/EventHelper.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DiscordBot.DataAccess;
+using DiscordBot.DataAccess.Exceptions;
+using DiscordBot.DataAccess.Models;
+using DSharpPlus.Entities;
+
+namespace DiscordBot.Commands
+{
+    class EventHelper
+    {
+        public static async Task<DiscordEvent?> GetEventFromMessageIdOrDefaultAsync(ulong messageId, IEventsSheetsService eventsSheetsService)
+        {
+            try
+            {
+                return await eventsSheetsService.GetEventFromMessageIdAsync(messageId);
+            }
+            catch (EventNotFoundException)
+            {
+                return null;
+            }
+        }
+
+        public static Dictionary<ulong, IEnumerable<EventResponse>> GetResponsesByUser(
+            Dictionary<EventResponse, IEnumerable<ulong>> signupsByResponse)
```

Comment:
One argument per line

Reply:
There's only one argument, should I move the method name to the next line?

Reply:
Ah sorry, my mistake! Clearly struggling to read properly this afternoon 🤦